lib/core/src/lib/form/components/widgets/date-time/date-time.widget.ts

```diff
@@ -42,68 +43,115 @@ import { WidgetComponent } from '../widget.component';
     ],
     templateUrl: './date-time.widget.html',
     styleUrls: ['./date-time.widget.scss'],
-    imports: [NgIf, TranslateModule, MatFormFieldModule, MatInputModule, MatDatetimepickerModule, FormsModule, ErrorWidgetComponent],
+    imports: [NgIf, TranslateModule, MatFormFieldModule, MatInputModule, MatDatetimepickerModule, ReactiveFormsModule, ErrorWidgetComponent],
     encapsulation: ViewEncapsulation.None
 })
-export class DateTimeWidgetComponent extends WidgetComponent implements OnInit {
+export class DateTimeWidgetComponent extends WidgetComponent implements OnInit, OnDestroy {
     minDate: Date;
     maxDate: Date;
+    datetimeInputControl: FormControl<Date>;
 
-    @Input()
-    value: any = null;
+    private datetimeChangesSubscription: Subscription;
```

Comment:
why not do takeUntil(this.onDestroy$) instead? Moreover this also should be either initiated with value or declared with `?`

Reply:
replaced by onDestroy subject